platform/machine/unprivqemu/cluster.go

```diff
@@ -101,7 +139,9 @@ func (qc *Cluster) NewMachineWithOptions(userdata *conf.UserData, options platfo
 
 	qc.mu.Lock()
 
-	qmCmd = append(qmCmd, "-netdev", "user,id=eth0,hostfwd=tcp:127.0.0.1:0-:22", "-device", platform.Virtio(qc.flight.opts.Board, "net", "netdev=eth0"))
+	sharedNetDev := "socket,id=shared0,mcast=230.0.0.1:" + strings.ReplaceAll(qc.mcastPortHolder.Addr().String(), "127.0.0.99:", "")
```

Comment:
We could split a bit this line to get a better reading:
```suggestion
        _, port, err := net.SplitHostPort(listener.Addr().String())
	if err != nil { ... }
	sharedNetDev := "socket,id=shared0,mcast=230.0.0.1:" + port
```

Reply:
I've simplified it a bit now